servers/rendering/renderer_viewport.cpp

```diff
@@ -151,10 +164,14 @@ void RendererViewport::_configure_3d_render_buffers(Viewport *p_viewport) {
 					render_height = height;
 					break;
 				case RS::VIEWPORT_SCALING_3D_MODE_FSR:
+				case RS::VIEWPORT_SCALING_3D_MODE_FSR2:
 					width = p_viewport->size.width;
 					height = p_viewport->size.height;
 					render_width = MAX(width * scaling_3d_scale, 1.0); // width / (width * scaling)
 					render_height = MAX(height * scaling_3d_scale, 1.0);
+
+					// Implementation has been copied from ffxFsr2GetJitterPhaseCount.
+					jitter_phase_count = uint32_t(8.0f * pow(float(width) / render_width, 2.0f));
```

Comment:
This adds jitter to regular FSR. Since regular FSR doesn't have a resolve pass it is super noticable. We need to force this to 0 with FSR1